supported_parts.md

```diff
@@ -122,6 +122,7 @@
 - ADG2128
 - ADIS16460
 - ADIS16475
+- ADIS16480
```

Comment:
Please include these:
           "adis16375",
            "adis16480",
            "adis16485",
            "adis16488",
            "adis16490",
            "adis16495-1",
            "adis16495-2",
            "adis16495-3",
            "adis16497-1",
            "adis16497-2",
            "adis16497-3"

Reply:
Already updated the supported parts. Thanks! :)